project/src/main/resources/templates/fragments/header.html

```diff
@@ -21,11 +21,18 @@
                 Cart <span id="cart_num" th:text="${num_in_cart}"></span>
               </a>
             </li>
+            <!-- KEEP COMMENTED UNTIL HAPPY WITH HOW CHECKOUT WORKS...
```

Comment:
What do you guys think about not having a checkout section in the header and have the only way to access the checkout page be from the cart page?

Reply:
I think this is what we had decided on ya. I think you mentioned you only had the checkout link the header so that it was easy to access in testing?